examples/import-database-flyway-maven/README.adoc

```diff
@@ -0,0 +1,122 @@
+= Import Database By Flyway By Maven Plugin
+:index-group: Import
+:jbake-type: page
+:jbake-status: not published/unrevised
+
+This is an example of how to use the Flyway tool to import a database via the Maven Plugin.
+
+[discrete]
+==== Import Database By Flyway By Maven Plugin
+
+To use Flyway, when building the application, we will use the Maven Plugin to import the database, we will use an in-memory database.
+
+The SQL instructions defined in the script will be used `V1_0__import-ImportSqlScriptTest.sql`:
+
+[source,sql]
+----
+CREATE TABLE table_test (ID INT NOT NULL, DESCRIPTION VARCHAR(120) NOT NULL, PRIMARY KEY (ID));
+insert into table_test (id, description) values(1, 'Test 1');
+insert into table_test (id, description) values(2, 'Test 2');
+insert into table_test (id, description) values(3, 'Test 3');
+----
+
+== Running the import by the Maven Plugin:
+
+The following steps must be followed:
+
+1 - Run the build of the entire project, in the project root directory, tomee:
+
+[source,bash]
+----
+-Pquick -Dsurefire.useFile=false -DdisableXmlReport=true -DuniqueVersion=false -ff -Dassemble -DskipTests -DfailIfNoTests=false clean install
```

Comment:
Markus, i read your comment, and I'm going to remove the dependencies from the main Tomee project, so that this project can be independent. And then it won't be necessary to run the Tomee project first.